sdk-docs/reference/flask.md

```diff
@@ -0,0 +1,53 @@
+# Integrations
+
+The flask charms produced by PaaS App Charmer support the following
+integrations:
```

Comment:
This section is a little confusing as you're saying it supports the following integrations, but then listing charms (and in one case a bundle of charms). Maybe we should say "support integrations with the following charms and bundles"?